v2/cy/cy_holidays_test.go

```diff
@@ -0,0 +1,100 @@
+// (c) Rick Arnold. Licensed under the BSD license (see LICENSE).
+
+package cy
+
+import (
+	"testing"
+	"time"
+
+	"github.com/rickar/cal/v2"
+)
+
+/*
```

Comment:
Looks like a temp comment that should be removed?

Reply:
Yup, debugging comment. Was removed.